packages/angular/cli/models/schematic-command.ts

```diff
@@ -61,7 +61,7 @@ export interface BaseSchematicSchema {
 export interface RunSchematicOptions extends BaseSchematicSchema {
   collectionName: string;
   schematicName: string;
-
+  additionalOptions?: { [key: string]: {} };
```

Comment:
Nit: Why not `Record<string, {}>`, It's not the first time I've seen this.

Reply:
Habit more than anything else; `Record` is fairly new.  I also tend to group `Record` in with `Pick`/`Extract` as more of type _reconstruction_ operators than anything else.  `Record` is very interesting in combination with string enums or string literal types.

In this case it probably is cleaner to use it though.